utility/path.cpp

```diff
@@ -90,4 +91,21 @@ fs::path lexically_relative(const fs::path &path, const fs::path &base)
 
 #endif
 
+void rename(const fs::path& oldPath, const fs::path& newPath)
+{
+#ifdef _WIN32
+    if (oldPath != newPath)
+    {
+        // use copy/delete instead of problematic rename
+        auto copyOptions(fs::copy_options::recursive
+                         | fs::copy_options::overwrite_existing
+                         | fs::copy_options::copy_symlinks);
+        copy(oldPath, newPath, copyOptions);
+        remove_all(oldPath);
```

Comment:
Does this work with symlinks as intented?

Reply:
It should, as `fs::copy_options::copy_symlinks` is specified in copy options.

Reply:
and for `remove_all`: Symlinks are not followed (symlink is removed, not its target).